qcfractal/cli/qcfractal_server.py

```diff
@@ -3,80 +3,188 @@
-    security.add_argument("--disable-ssl", type=bool, default=False, help="Disables SSL if present, if False a SSL cert will be created for you")
-    security.add_argument("--tls-cert", type=str, default=None, help="Certificate file for TLS (in PEM format)")
-    security.add_argument("--tls-key", type=str, default=None, help="Private key file for TLS (in PEM format)")
-
-    general = parser.add_argument_group('General Settings')
-    general.add_argument("database_name", type=str, help="The name of the database to use with the storage socket")
-    general.add_argument("--server-name", type=str, default="QCFractal Server", help="The server name to broadcast")
-    general.add_argument("--query-limit", type=int, default=1000, help="The maximum query size to server")
-    general.add_argument("--log-prefix", type=str, default=None, help="The logfile prefix to use")
-    general.add_argument("--database-uri", type=str, default="mongodb://localhost", help="The database URI to use")
-    general.add_argument("--port", type=int, default=7777, help="The server port")
-    general.add_argument("--compress-response", type=bool, default=True, help="Compress the response or not")
-    general.add_argument("--config-file", type=str, default=None, help="A configuration file to use")
-    general.add_argument("--start-periodics", type=bool, default=True, help="Start the periodic calls or not, always recommended unless running fractal-server behind a proxy")
-
-    parser._action_groups.reverse()
+    fractal_args = start.add_argument_group('Server Settings')
+    # Allow port and logfile to be altered on the fly
+    for field in ["port", "logfile"]:
+        cli_name = "--" + field.replace("_", "-")
+        fractal_args.add_argument(cli_name, **FractalServerSettings.help_info(field))
 
+    fractal_args.add_argument("--database-name",
+                              default=None,
+                              type=str,
+                              help="The database to connect to, defaults to the default database name.")
+    fractal_args.add_argument("--server-name", **FractalServerSettings.help_info("name"))
+    fractal_args.add_argument(
+        "--start-periodics",
+        default=True,
+        type=bool,
+        help="Expert! Can disable periodic update (services, heartbeats) if False. Useful when running behind a proxy."
+    )
+
+    fractal_args.add_argument("--disable_ssl",
+                              default=False,
+                              type=bool,
+                              help="Disables SSL if present, if False a SSL cert will be created for you.")
+    fractal_args.add_argument("--tls-cert", type=str, default=None, help="Certificate file for TLS (in PEM format)")
+    fractal_args.add_argument("--tls-key", type=str, default=None, help="Private key file for TLS (in PEM format)")
+
+    compute_args = start.add_argument_group('Local Computation Settings')
+    compute_args.add_argument("--local-manager",
```

Comment:
I don't quite follow. The parameter provided should be the number of processes to spin up.

Reply:
But the number is optional, you can specify `--local-manager` with no args or it could be specified with an int `--local-manager 4`, I'm just worried it might get confusing, but there may not be conditions where that is ambiguous with the rest of the commands.